modules/ROOT/pages/deploy-maven-3.x.adoc

```diff
@@ -81,67 +88,69 @@ This example uses placeholder values. Configure each value with values specific
 
 == Redeploy to Runtime Fabric
 
-To redeploy the application, run the same command as you did to deploy. Runtime Fabric rewrites the application you had deployed.
+To redeploy the application, run the same command you ran to deploy it. Runtime Fabric redeploys the 
+application.
 
 == Runtime Fabric Deployment Reference
 
 [%header%autowidth.spread,cols="a,a"]
 |===
 |Parameter | Description
-|runtimeFabricDeployment | specifies a top-level element.
-| uri | specifies your Anypoint Platform URI. +
+|runtimeFabricDeployment | Specifies a top-level element.
+| uri | Specifies your Anypoint Platform URI. +
 If not set, the default value is `+https://anypoint.mulesoft.com+`.
-| muleVersion | specifies the Mule runtime version installed in your Runtime Fabric instance. +
-Ensure that this value is at equal or higher than the minimum required Mule runtime version of your application.
-| username | specifies your Anypoint Platform username.
-| password | specifies your Anypoint Platform password.
-| applicationName | specifies the name of your application deployed in Exchange. +
-This name is part of the domain of your deployed app.
-| businessGroup (optional) | specifies the business group where you want to deploy the application. If not defined, name of the master organization is used as the default.
+| muleVersion | Specifies the Mule runtime version installed in your Runtime Fabric instance. +
+Ensure that this value is equal to or later than the minimum required Mule runtime version of your application.
```

Comment:
"Equal to" is a bit confusing in this context.

Reply:
Fixed - thank you!